### DISCUSSION TITLE
INPUT ERROR: Element type not properly defined. What happened?

### DISCUSSION THREAD

**Discussion by zwlazyt** [Anura3D/Anura3D_OpenSource#125] [Q&A]
When I use Anura3D to generate the file, a window named 'GID Application Error' pops up showing:
INPUT ERROR: Element type not properly defined. Only the following element types are supported: triangular 3-noded, tetrahedral 10-noded.
INPUT ERROR: Element type not properly defined. Only the following element types are supported: triangular 3-noded, tetrahedral 10-noded.
    while executing
"error [= "INPUT ERROR: Element type not properly defined. Only the following element types are supported: triangular 3-noded, tetrahedral 10-noded."]"
    (procedure "Anura3D::WriteCalculationFile_GOM" line 35)
    invoked from within
"Anura3D::WriteCalculationFile_GOM $GOM_name"
    (procedure "Anura3D::Calculate" line 21)
    invoked from within
"Anura3D::Calculate"
    (menu invoke)

I don't understand why this is happening.This is all done with the use of a unstructured mesh for this：

![屏幕截图 2023-08-23 115105](https://github.com/Anura3D/Anura3D_OpenSource/assets/44886118/56d208ef-e7e2-407e-a99f-f52a9c2adae5)

**Comment by Armin-Geranghadr** [comment]
Hello,

Can you solve this error? Because I am dealing with it now and I hope you can give me some clues to solve it.

Thanks
